src/module.c

```diff
@@ -6585,7 +6585,8 @@ moduleType *moduleTypeLookupModuleByName(const char *name) {
         listRewind(module->types,&li);
         while((ln = listNext(&li))) {
             moduleType *mt = ln->value;
-            if (memcmp(name,mt->name,sizeof(mt->name)) == 0) {
+            if ((!case_flag && memcmp(name,mt->name,sizeof(mt->name)) == 0)
+                ||(case_flag && !strcasecmp(name, mt->name))) {
```

Comment:
```suggestion
                || (case_flag && !strcasecmp(name, mt->name))) {
```

Reply:
hi, I just added a space and didn't put brace to the new line, not sure if my coding style is right.

Reply:
because there are multiple lines in the `if`, redis style recommends to put the brace on a single line.

Reply:
Thank you for your detailed answer, I will follow the style